Update oauth docs (#745)
team-alembic/ash_authentication: documentation/tutorials/github.md

end
```

> Ensure you set the `hashed_password` to null in case you have one.

Reviewer: ```suggestion
Ensure you set the `hashed_password` to `allow_nil?` if you are also using the password strategy.
```

end
```

> Ensure you set the `hashed_password` to `allow_nil?` if you are also using the password strategy.

Author: @zachdaniel It's actually under the Get Started Page though: https://ash-hq.org/docs/guides/ash_authentication/latest/tutorials/get-started#example

Reviewer: Not sure I understand what you mean. The getting started guide has you using the password strategy.
